doc/user/content/transform-data/patterns/durable-subscriptions.md

```diff
@@ -80,46 +172,28 @@ object will lead to increased resource utilization in Materialize.
     for that cluster to go up, which might require you to size up your cluster
     and consume additional compute credits.
 
-### Clean-up
+#### Best practices
 
-The history retention period represents the minimum amount of historical data
```

Comment:
What's the motivation to cut the details on clean-up? AFAIR it was included because a user asked about how history cleanup worked.

Reply:
Not cut.  Just moved up to the Behavior/Semantics section 
 https://github.com/MaterializeInc/materialize/pull/28646/files#diff-cf72fc8c3a01d715512ded2b51842cf0cb083481d0065b3afc379f00230e9889R74

Reply:
Sounds good.  Will move it there in the next patch of this PR.